ci/common.sh

```diff
@@ -75,6 +75,15 @@ install_curl () {
     curl -V
 }
 
+install_static_curl () {
```

Comment:
if this is not going to be used anywhere else, better move it in quic.yaml.

Reply:
OK, will do